### ISSUE TITLE
Plans on moving Vector Layer Code for OL 3.5?

### ISSUE THREAD

**Issue by milafrerichs** [tombatossals/angular-openlayers-directive#120]
Hey thanks for merging my PR and for the directive.

Do you have plans or would accept a PR for moving to the new Vector API in OL3 3.5? (https://github.com/openlayers/ol3/releases/tag/v3.5.0)

```js
var source = new ol.source.Vector({
  url: 'features.json',
  format: new ol.format.GeoJSON()
});
```

I would start working on it if you would consider merging it.
Thanks.

**Comment by tombatossals** [comment]
Yes of course, it would be great if you could code the adaption to the new Vector API, I'm always trying to be in sync with the latest OpenLayers version.

In the meantime, I will update the library to the v3.5.0 to test the actual functionality is working.

Thanks

**Comment by tombatossals** [comment]
Hi, I have created this branch to update the Vector creation calls to the new API:

https://github.com/tombatossals/angular-openlayers-directive/tree/v3.5.0

**Comment by tombatossals** [comment]
I have the v3.5.0 branch working with the ol3 v3.5.0 branch. I will merge this branch on master in a few days, releasing a new version so people using ol3 < v3.5.0 can continue using their code.

**Comment by juristr** [comment]
@tombatossals great work! I was just looking into this issue. Are there any news on release of this as I'd like to upgrade ol3 to v3.6.

Btw, what version do you plan to release this with? Being somehow a breaking change this would be an upgrade of the major version, right (to stay in synch with semver)?

**Comment by juristr** [comment]
@tombatossals Any updates here? I'm sorry to ask again, but I'd have to know at which point you are as we'd need some new features from v3.6 which currently isn't compatible with this directive, though. :worried: 

Thx for any info on your end. :smile:

**Comment by juristr** [comment]
@tombatossals I upgraded your v3.5.0 branch with the latest changes in angular-openlayers-directive and I'm using it now with ol v.3.6 for a while. Seems quite stable. My fork is here: https://github.com/juristr/angular-openlayers-directive/tree/v3.5.0

If you want, we could organize a PR to upgrade? What are your plans about this? I'd like to through my custom fork away and continue with your repo again. :smile:

**Comment by juristr** [comment]
Closing this as it will be solved soon with #155 .

**Comment by juristr** [comment]
Ok, `v1.0.0` released: https://github.com/tombatossals/angular-openlayers-directive/releases/tag/v1.0.0
Now official compatible with ol3 > v3.5
